File: serializacion/Serializacion.dbase/Model/Requerimiento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    [Serializable]
    public class Requerimiento
    {
        [NonSerialized]
        int _numero;

        string _descripcion;

        List<Tarea> _tareas = new List<Tarea>();

        public Requerimiento(int pNumero, string pDescripcion)
        {
            _numero = pNumero;
            _descripcion = pDescripcion;
        }

        public int Numero
        {
            get
            {
                return _numero;
            }
            set
            {
                _numero = value;
            }
        }

        public string Descripcion
        {
            get
            {
                return _descripcion;
            }
            set
            {
                _descripcion = value;
            }
        }

        public void AddTarea(Tarea unaTarea)
        {
            _tareas.Add(unaTarea);
        }

        public List<Tarea> GetTareas()
        {
            return _tareas;
        }
    }
}

File: serializacion/Serializacion.dbase/Modelo/Requerimiento.vb
Imports System.Collections.Generic

<Serializable()> _
Public Class Requerimiento

    Private _numero As Integer
    Private _descripcion As String
    Private _tareas As New List(Of Tarea)


    Public Sub New(ByVal pNumero As Integer, ByVal pDescripcion As String)

        _numero = pNumero
        _descripcion = pDescripcion

    End Sub

    Public Property Numero() As Integer
        Get
            Return _numero
        End Get
        Set(ByVal value As Integer)
            _numero = value
        End Set
    End Property

    Public Property Descripcion() As String
        Get
            Return _descripcion
        End Get
        Set(ByVal value As String)
            _descripcion = value
        End Set
    End Property

    Public Sub AddTarea(ByVal unaTarea As Tarea)
        _tareas.Add(unaTarea)
    End Sub

    Public Function GetTareas() As List(Of Tarea)
        Return _tareas
    End Function

End Class
